scripts/test_api_calls.py

```diff
@@ -0,0 +1,145 @@
+     node_info = {"local_ip":str(key), "mac_address":str(value), "node_id":node_id[key_index], "node_name":node_name, "server_port":8080, "veth":"eth0"}
+     data["host_info"] = node_info
+     post_httprequest(url, data)
+
+
+def create_vpc(port):
+  network = {}
+  url = 'http://localhost:{}/project/{}/vpcs'.format(port, project_id)
+  network_info = {"admin_state_up":True, "revision_number":0, "cidr":"10.0.1.0/24", "default":True, "description":"vpc", "dns_domain":"domain", "id":network_id, "is_default":True, "mtu":1400, "name":"sample_vpc", "port_security_enabled":True, "project_id":project_id}
+  network["network"]=network_info
+  post_httprequest(url, network)
+
+
+def create_subnet(port):
+  subnet = {}
+  url = 'http://localhost:{}/project/{}/subnets'.format(port, project_id)
+  subnet_info = { "cidr":"10.0.1.0/24", "id":subnet_id, "ip_version":4, "network_id":network_id, "name":"subnet1", "host_routes":[{"destination":"10.0.1.0/24","nexthop":"10.0.1.1"} ] }
+  subnet["subnet"] = subnet_info 
+  post_httprequest(url, subnet)
+
+def create_security_groups(port):
+  security_groups ={}
+  url = 'http://localhost:{}/project/{}/security-groups'.format(port, project_id)
+  security_group_info = {"create_at":"string","description":"string","id":security_group_id,"name":"sg1","project_id":project_id,"security_group_rules":[],"tenant_id":tenant_id,"update_at":"string"}
+  security_groups["security_group"]=security_group_info
+  post_httprequest(url,security_groups)
+
+
+def get_subnets(port):
+  url = 'http://localhost:{}/project/{}/subnets'.format(port, project_id)
+  print(get_httprequest(url))
+
+
+def get_nodes(port):
+  url= 'http://localhost:{}/nodes'.format(port)
+  print(get_httprequest(url))
+
+
+def get_vpcs(port):
+  url= 'http://localhost:{}/project/{}/vpcs'.format(port,project_id)
+  print(get_httprequest(url))
+
+def create_ports(port,ip_addrs,port_id):
+  url= 'http://localhost:{}/project/{}/ports'.format(port,project_id)
+  for index in range(len(ip_addrs)):
+   ports ={}
+   port_info= {"admin_state_up":True,"allowed_address_pairs":[{"ip_address":"11.11.11.11","mac_address":"00-AA-BB-15-EB-3F"}],"binding:host_id":"node1","binding:vif_details":{},"create_at":"string","description": "string","device_id":device_id,"device_owner": "compute:nova","dns_assignment": {},"dns_domain": "string","dns_name": "string","extra_dhcp_opts": [{"ip_version": "string","opt_name":"string","opt_value": "string"}],"fast_path": True,"fixed_ips":[{"ip_address": ip_addrs[index],"subnet_id":subnet_id}],"id": port_id[index],"mac_learning_enabled": True,"name": "port{}.format(index)","network_id": network_id,"network_ns": "string","port_security_enabled": True,"project_id": project_id,"qos_network_policy_id": "string","qos_policy_id": "string","revision_number": 0,"security_groups": [security_group_id],"tags": ["string"],"tenant_id":tenant_id,"update_at": "string","uplink_status_propagation": True,"veth_name":"string"}
```

Comment:
@kiran1048 Port is the most important entity, we may need to add or modify some fields in the port entity. It's better to read the port configuration from files or have a flexible way to read some field's value dynamically.